deploy/helm/prometheus-overrides.yaml

```diff
@@ -207,7 +207,7 @@ prometheus:
       # scheduler_e2e_scheduling_latency_microseconds
```

Comment:
@perk-sumo I think these metrics also got changed from `scheduler_e2e_scheduling_latency_microseconds` to `scheduler_e2e_scheduling_latency_seconds`

Reply:
Actually it changed to `e2e_scheduling_duration_seconds`
https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/metrics/metrics.go#L74